### PR TITLE
Update qBittorrent documentation - add get_torrents and get_all_torrents services

### PR DESCRIPTION
**Pull request by finder39**
## Proposed change
Updating the documentation for the QBittorrent integration for changes made in a pull request on core.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/105781
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/qbittorrent.markdown [hunk 1]
```diff
@@ -34,3 +34,42 @@ The qBittorrent integration will add the following sensors:
...
+| `torrent_filter`       | no       | The type of torrents you want in the response  | all, active, inactive, paused, downloading, seeding |
+
+```yaml
+service: qbittorrent.get_torrents
+data:
+  filter: "all"
+response_variable: torrents
+```
+
+The response data contains the field `torrents` which contains a dictionary of torrents. The name of the torrents are the keys.
```
Comment (frenck) [inline source/_integrations/qbittorrent.markdown]:
```suggestion
The response data contains the field `torrents` which contains a dictionary of torrents. The names of the torrents are the keys.
```

File: source/_integrations/qbittorrent.markdown [hunk 2]
```diff
@@ -34,3 +34,42 @@ The qBittorrent integration will add the following sensors:
...
+| `torrent_filter`       | no       | The type of torrents you want in the response | all, active, inactive, paused, downloading, seeding |
+
+```yaml
+service: qbittorrent.get_all_torrents
+data:
+  filter: "all"
+response_variable: all_torrents
+```
+
+The response data contains the field `all_torrents` which contains a dictionary of integrations, which each contain an dictionary of torrents. The name of the torrents are the keys.
```
Comment (frenck) [inline source/_integrations/qbittorrent.markdown]:
```suggestion
The response data contains the field `all_torrents`, which contains a dictionary of integrations, which each contains a dictionary of torrents. The names of the torrents are the keys.
```

**Review by frenck — APPROVED** [review]
Thanks, @finder39 👍 

../Frenck

**Comment by finder39** [comment]
@frenck thanks for the spelling/grammar corrections 🙃

**Pull request merged**